client/state/initial-state.js

```diff
@@ -48,29 +48,26 @@ function deserialize( state ) {
 }
 
 /**
- * Determines whether to add "sympathy" by randomly clearing out persistent
- * browser state and loading without it
+ * Determines whether to clear persistent state before a given page load.
  *
- * Can be overridden on the command-line with two flags:
- *   - ENABLE_FEATURES=force-sympathy npm start (always sympathize)
- *   - ENABLE_FEATURES=no-force-sympathy npm start (always prevent sympathy)
+ * Can be overridden on the command line with two flags:
+ *   - ENABLE_FEATURES=always-clear-persistent-state npm start
+ *   - ENABLE_FEATURES=never-clear-persistent-state npm start
```

Comment:
although I mentioned that I had possibly named these poorly, we will want to be careful or at least check with/announce to others that this is changing. I think some people get around the intention of this feature by forcing it off in scripts.

of course, if we pull out the rug from under them it would likely only lead to more sympathy for customer frustration when their sympathy-blocker stops working :wink: